Remotion/Web/Core/UI/Controls/SmartLabel.cs

```diff
@@ -119,7 +120,7 @@ public string GetText()
         text = "[Label for " + ForControl + "]";
     }
     ForControl = forControlBackUp;
-    return text;
+    return WebString.CreateFromText (text);
```

Comment:
One of the next tasks will change the returntype of smartControl.DisplayName to WebString (line 118). We can either refactor now to allow easier substitution or the type or later when the next task gets done.

From what I'm reading it will either be to refactor into return statements for text or individual wrapping of the text results. Either way, line 123 will be changed again, either back to the original form or to be deleted entirely. 

You choice when we do this.

Reply:
I think that is better done when the time comes instead of digging up this feature branch again.